static-crawling/static-crawling_assignment.py

```
@@ -0,0 +1,176 @@
+
+from bs4 import BeautifulSoup
+import requests
+"""
+TODO
+1. requests 라이브러리로 url을 받아옵니다.
+2. Beautifulsoup로 html 문서를 파싱합니다.
+"""
+
+# 알라딘 베스트셀러 페이지 URL
+url = "https://www.aladin.co.kr/shop/common/wbest.aspx?BestType=Bestseller&BranchType=1&CID=0&page=1&cnt=1000&SortOrder=1"
+response = requests.get(url)  # 요청 보내기
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)